plugins/newrelic/named_actions.yaml

```
@@ -1,8 +1,18 @@
 connection_type:
   name: newrelic
   attributes:
+    New-Relic-Server:
+      description: New Relic API address. Most users can use the default url, if you are
+        using a New Relic account which requries a different server (i.e Europe region account),
+        you will need to change this field.
+      display_name: API Address
+      field_type: 0
+      index: 1
+      input_type: text
+      placeholder: 'New Relic API URL. Default: https://api.newrelic.com'
+      required: false
```

```suggestion
      display_name: API Address
      field_type: 2
      index: 1
      input_type: text
      options:
        - https://api.newrelic.com
        - https://api.eu.newrelic.com
      required: true
```
Unless there are more than 2 options, let's use a select element instead free text